### DISCUSSION TITLE
Allow to the powerShare feature be able to be configured per mode

### DISCUSSION THREAD

**Discussion by lenocid** [ikemen-engine/Ikemen-GO#925] [Feature Request]

**Comment by Sluigi123** [comment]
If set to ON: One team has one Power Meter that they share.
If set to OFF: Each player has their own separate Power Meter, regardless of them being in a team or not; taking reference to Street Fighter EX3's Tag/Team System.

Did I get this right? 
If so, it sounds doable, but it's going to be tedious for the dev team.

**Comment by lenocid** [comment]
> If set to ON: One team has one Power Meter that they share. If set to OFF: Each player has their own separate Power Meter, regardless of them being in a team or not; taking reference to Street Fighter EX3's Tag/Team System.
> 
> Did I get this right? If so, it sounds doable, but it's going to be tedious for the dev team.

But my point is between simul and tag mode,the teampowershare switch is already had.If this switch can be more elaborate,it can save yourself some trouble.

**Comment by Windblade-GR01** [comment]
In other words.
Allow to the powerShare feature be able to be configured per mode.
